install/kubernetes/helm/istio/values.yaml

```diff
@@ -333,7 +337,7 @@ global:
 
   # The trust domain corresponds to the trust root of a system
   # Refer to https://github.com/spiffe/spiffe/blob/master/standards/SPIFFE-ID.md#21-trust-domain
-  trustDomain: ""
+  trustDomain: "cluster.local"
```

Comment:
Don't we already have cluster.local configuration ? What's different from 'domain' and 'trustDomain' ?

Reply:
It's a trust domain as per https://github.com/spiffe/spiffe/blob/master/standards/SPIFFE-ID.md#21-trust-domain

So the context it's used in is tls certificates, not networks or dns or the like.